pkg/web_app/lib/src/account.dart

```diff
@@ -231,7 +231,7 @@ class _PkgAdminWidget {
       return false;
     }
 
-    await rpc(
+    await rpc<void>(
```

Comment:
This won't be inferred?

Reply:
Not sure why, but the analyzer gave a hard error if we leave `<void>` out:
```
Couldn't infer type parameter 'R'.

Tried to infer 'Null' for 'R' which doesn't work:
  Parameter 'onSuccess' declared as     'FutureOr<void> Function(R?)?'
                        but argument is 'void Function(Object?)'.
The type 'Null' was inferred from:
  Parameter 'fn' declared as     'Future<R?> Function()?'
                 but argument is 'Future<Null> Function()'.

Consider passing explicit type argument(s) to the generic.
```